apps/journeys-admin/src/components/Editor/Slider/Settings/CanvasDetails/Footer/Footer.stories.tsx

```diff
@@ -65,20 +53,6 @@ const Template: StoryObj<
   }
 }
 
-export const Default = { ...Template, args: { hostTitle: null } }
-
-export const Filled = {
-  ...Template,
-  args: {
-    hostTitle: `John Geronimo "The Rock" Johnson`,
-    chatButtons: [
-      {
-        id: '1',
-        link: 'https://m.me/user',
-        platform: ChatPlatform.facebook
-      }
-    ]
-  }
-}
+export const Default = { ...Template, args: { host: null } }
```

Comment:
I think you can remove since, since your not actually using any `args` when you render `Template`
```suggestion
export const Default = { ...Template }
```